production/ksonnet/README.md

```diff
@@ -54,19 +57,36 @@ promtail + {
 ```
 Notice that `container_root_path` is your own data root for docker daemon, use `docker info | grep "Root Dir"` to get it.
 
-Then do `ks show loki` to see the manifests that'll be deployed to your cluster.
-Apply them using `ks apply loki`.
+As a last step, fill add the correct `spec.apiServer` and `spec.namespace` to `environments/default/spec.json`:
+
+{
+  "apiVersion": "tanka.dev/v1alpha1",
+  "kind": "Environment",
+  "spec": {
+    "apiServer": "https://localhost:6443",
+    "namespace": "default"
+  }
+}
+
+Now use `tk show environments/default` to see the yaml, and `tk apply environments/default` to apply it to the cluster.
 
 ## Deploying Loki to your cluster.
 
 If you want to further also deploy the server to the cluster, then run the following to install the module:
 
 ```
-jb install github.com/grafana/loki/production/ksonnet/loki
+# create a directory (any name works)
```

Comment:
basically the same as above, we don't need to repeat that here.

Rather treat this as a follow up to the above, only adding Loki to the deployment as well.